src/BinaryParsers/ElfBinary/Dwarf/DwarfCompileCommandLineInfo.cs

```diff
@@ -14,5 +14,10 @@ public class DwarfCompileCommandLineInfo
         public string CommandLine { get; set; }
         public DwarfLanguage Language { get; set; }
         public string CompileDirectory { get; set; }
+
+        public override string ToString()
+        {
+            return "Module: " + FileName + ", CommandLine: " + CommandLine;
```

Comment:
>CommandLine: " + CommandLine [](http://example.com/codeflow?start=46&length=28)

I would not add the commandline. It can be very big and for the end-user, that isn't that important, since we are already pointing the module

Reply:
sounds good, since each line now will have only 1 field, I will remove the new line, it will be:
file1.c, file2.c